src/main/router.js

```diff
@@ -73,6 +73,9 @@ const Blog = React.lazy(() => import(/* webpackChunkName: "blog" */ '../scenes/b
 const Notfound = React.lazy(() =>
   import(/* webpackChunkName: "not-found" */ './../styled_scenes/NotFound'),
 );
+const AdminModeration = React.lazy(() =>
+  import(/* webpackChunkName: "moderation" */ '../scenes/admin-moderation'),
```

Comment:
Could you please wrap the `React.lazy` with `asyncCommonHOCs` instead of doing that in line 163? In a branch that will be merged soon the router is now a component and that would cause it to create a new instance on each render.